### PR TITLE
Replace "Consent" with "Access"

### PR DESCRIPTION
**Pull request by solid-akb**
## Description

We decided to use the word "Access" instead of "Consent" when discussing Agent/Resource Access. This PR removes "Consent" from the application. 

## Changes
Remove any text related to "Consent" in PodBrowser
## Commit checklist

- [ ] All acceptance criteria are met.
- [ ] Includes tests to ensure functionality and prevent regressions.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.

### CODE REVIEW

File: __testUtils/mockAccessRequestDetails.js [hunk 1]
```diff
@@ -19,43 +19,43 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-export function getConsentRequestDetailsOnePurpose() {
+export function getAccessRequestDetailsOnePurpose() {
   return {
     "@context": [
       "https://www.w3.org/2018/credentials/v1",
       "https://w3id.org/security/suites/ed25519-2020/v1",
-      "https://consent.pod.inrupt.com/credentials/v1",
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:27]: these shouldn't change

File: __testUtils/mockAccessRequestDetails.js [hunk 2]
```diff
@@ -19,43 +19,43 @@
...
+      "https://access.pod.inrupt.com/credentials/v1",
     ],
-    id: "https://consent.pod.inrupt.com/vc/53973727-f4d0-9e8dbdc041fd",
-    type: ["VerifiableCredential", "SolidCredential", "SolidConsentRequest"],
-    issuer: "https://consent.pod.inrupt.com",
+    id: "https://access.pod.inrupt.com/vc/53973727-f4d0-9e8dbdc041fd",
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:29]: these shouldn't change

Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:29]: these shouldn't change

File: __testUtils/mockAccessRequestDetails.js [hunk 3]
```diff
@@ -19,43 +19,43 @@
...
+    type: ["VerifiableCredential", "SolidCredential", "SolidAccessRequest"],
+    issuer: "https://access.pod.inrupt.com",
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:31]: same here.. both URL and "SolidAccessRequest"

File: __testUtils/mockAccessRequestDetails.js [hunk 4]
```diff
@@ -19,43 +19,43 @@
...
     issuanceDate: "2021-05-26T16:40:03Z",
     expirationDate: "2022-06-23T16:40:03Z",
     credentialSubject: {
       id: "https://mockappurl.com",
       inbox: "https://mockappurl.com/inbox/",
-      hasConsent: [
+      hasAccess: [
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:37]: this shouldn't change either

File: __testUtils/mockAccessRequestDetails.js [hunk 5]
```diff
@@ -19,43 +19,43 @@
...
         {
           mode: ["Read", "Write"],
-          hasStatus: "ConsentStatusRequested",
+          hasStatus: "AccessStatusRequested",
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:40]: same

File: __testUtils/mockAccessRequestDetails.js [hunk 6]
```diff
@@ -19,43 +19,43 @@
...
           forPersonalData: ["https://pod.inrupt.com/alice/private/data/"],
           forPurpose: "https://example.com/SomeSpecificPurpose",
         },
         {
           mode: ["Read"],
-          hasStatus: "ConsentStatusRequested",
```
Comment (VirginiaBalseiro) [inline __testUtils/mockAccessRequestDetails.js:46]: same as others... in short we only want to change the PB components names and local variables, not anything related to the VC, and that includes mock VCs and the way we destructure/use it in the components

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
I commented a few instances where we shouldn't change consent > access, but in short: we only want to change the names of PB components and local variables, but nothing related to the VC, including mock VCs and the way we destructure/use it in the code, including the uses in the functions that extract info from the VC.

File: components/accessRequestForm/index.test.jsx [hunk 7]
```diff
@@ -224,19 +224,19 @@ describe("Consent Request Form", () => {
...
             agentWebId={agentWebId}
           />
-        </ConsentRequestContextProvider>
+        </AccessRequestContextProvider>
       </ConfirmationDialogProvider>
     );
     const purpose = getAllByTestId(TESTCAFE_ID_PURPOSE_CHECKBOX_INPUT)[0];
     userEvent.click(purpose);
-    const toggle = getAllByTestId(TESTCAFE_ID_CONSENT_ACCESS_SWITCH)[0];
+    const toggle = getAllByTestId(TESTCAFE_ID_ACCESS_ACCESS_SWITCH)[0];
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/index.test.jsx:237]: this looks weird now.. maybe just GRANT_ACCESS_SWITCH? same for the string value.

File: components/accessRequestForm/index.test.jsx [hunk 8]
```diff
@@ -251,22 +251,22 @@ describe("Consent Request Form", () => {
...
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/index.test.jsx:267]: same as above

File: components/accessRequestForm/requestSection/index.jsx [hunk 9]
```diff
@@ -445,7 +445,7 @@ export default function RequestSection(props) {
           <span className={bem("header__content")}>
             {`${agentName} ${getPolicyDetailFromAccess(
               modesObject,
-              "titleConsent"
+              "titleAccess"
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/requestSection/index.jsx:448]: let's name this `titleAccessGrant` (and change also in the function `getPolicyDetailFromAccess` if needed)

File: components/accessRequestForm/requestSection/index.test.jsx [hunk 10]
```diff
@@ -120,7 +120,7 @@ describe("Request Section", () => {
     const updatedSwitches = getAllByRole("checkbox", { checked: false });
 
     userEvent.click(updatedSwitches[0]);
-    expect(getByTestId("consent-access-switch")).toBeChecked();
+    expect(getByTestId("access-access-switch")).toBeChecked();
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/requestSection/index.test.jsx:123]: same as above

File: components/accessRequestForm/requestSection/index.test.jsx [hunk 11]
```diff
@@ -140,7 +140,7 @@ describe("Request Section", () => {
     });
     userEvent.click(switches[0]);
     await waitFor(() => {
-      expect(getByTestId("consent-access-switch")).toBeChecked();
+      expect(getByTestId("access-access-switch")).toBeChecked();
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/requestSection/index.test.jsx:143]: same as above

File: components/container/index.test.jsx [hunk 12]
```diff
@@ -47,7 +47,7 @@ import { TESTCAFE_ID_SPINNER } from "../spinner";
 import * as accessControlFns from "../../src/accessControl";
 import mockAccessControl from "../../__testUtils/mockAccessControl";
 import mockPermissionsContextProvider from "../../__testUtils/mockPermissionsContextProvider";
-import useConsentBasedAccessForResource from "../../src/hooks/useConsentBasedAccessForResource";
+import useAccessBasedAccessForResource from "../../src/hooks/useAccessBasedAccessForResource";
```
Comment (VirginiaBalseiro) [inline components/container/index.test.jsx:50]: let's call this `useAccessGrantBasedAccessForResource` - also rename the hook

File: components/container/index.test.jsx [hunk 13]
```diff
@@ -67,8 +67,8 @@ const mockedResourceInfoHook = useResourceInfo;
 jest.mock("../../src/hooks/useAccessControl");
 const mockedAccessControlHook = useAccessControl;
 
-jest.mock("../../src/hooks/useConsentBasedAccessForResource");
-const mockedUseConsentBasedAccessForResource = useConsentBasedAccessForResource;
+jest.mock("../../src/hooks/useAccessBasedAccessForResource");
+const mockedUseAccessBasedAccessForResource = useAccessBasedAccessForResource;
```
Comment (VirginiaBalseiro) [inline components/container/index.test.jsx:71]: same as above

File: components/container/index.test.jsx [hunk 14]
```diff
@@ -119,7 +119,7 @@ describe("Container view", () => {
       "https://myaccount.mypodserver.com/private",
       "https://myaccount.mypodserver.com/note.txt",
     ]);
-    mockedUseConsentBasedAccessForResource.mockReturnValue({ permissions: [] });
+    mockedUseAccessBasedAccessForResource.mockReturnValue({ permissions: [] });
```
Comment (VirginiaBalseiro) [inline components/container/index.test.jsx:122]: make sure to update here too

File: components/pages/privacy/access/requests/index.jsx [hunk 15]
```diff
@@ -30,39 +30,39 @@ import { createStyles, Typography, Link } from "@material-ui/core";
...
 } from "../../../../../__testUtils/mockApp";
-import { getRequestorWebId } from "../../../../../src/models/consent/request";
+import { getRequestorWebId } from "../../../../../src/models/access/request";
 import { getProfileResource } from "../../../../../src/solidClientHelpers/resource";
 import Spinner from "../../../../spinner";
 
 const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
-export default function ConsentShow() {
+export default function AccessShow() {
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/access/requests/index.jsx:47]: let's call it AccessRequestShow

File: components/pages/privacy/access/requests/index.test.jsx [hunk 16]
```diff
@@ -26,16 +26,16 @@ import { mockContainerFrom } from "@inrupt/solid-client";
 import * as resourceHelpers from "../../../../../src/solidClientHelpers/resource";
 import { renderWithTheme } from "../../../../../__testUtils/withTheme";
 import mockSessionContextProvider from "../../../../../__testUtils/mockSessionContextProvider";
-import ConsentPage from "./index";
-import getConsentRequestDetails from "../../../../../__testUtils/mockConsentRequestDetails";
+import AccessPage from "./index";
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/access/requests/index.test.jsx:29]: AccessRequestPage

File: components/pages/privacy/access/requests/index.test.jsx [hunk 17]
```diff
@@ -26,16 +26,16 @@ import { mockContainerFrom } from "@inrupt/solid-client";
...
+import getAccessRequestDetails from "../../../../../__testUtils/mockAccessRequestDetails";
 import { mockAppDataset } from "../../../../../__testUtils/mockApp";
 import useContainer from "../../../../../src/hooks/useContainer";
 import * as containerFns from "../../../../../src/models/container";
 
 jest.mock("../../../../../src/hooks/useContainer");
 const mockedUseContainer = useContainer;
 
-describe("Consent Page", () => {
+describe("Access Page", () => {
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/access/requests/index.test.jsx:38]: same as above

File: components/resourceDetails/index.test.jsx [hunk 18]
```diff
@@ -33,28 +33,28 @@ import ResourceDetails, {
...
 import useAccessControlType from "../../src/hooks/useAccessControlType";
 import { TESTCAFE_ID_AGENT_ACCESS_TABLE } from "./resourceSharing/agentAccessTable";
 
 const accessControl = mockAccessControl();
 const dataset = mockSolidDatasetFrom("http://example.com/container/");
 
-jest.mock("../../src/hooks/useConsentBasedAccessForResource");
-const mockedUseConsentBasedAccessForResource = useConsentBasedAccessForResource;
+jest.mock("../../src/hooks/useAccessBasedAccessForResource");
+const mockedUseAccessBasedAccessForResource = useAccessBasedAccessForResource;
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/index.test.jsx:44]: same as above

File: components/resourceDetails/index.test.jsx [hunk 19]
```diff
@@ -33,28 +33,28 @@ import ResourceDetails, {
 import mockAccessControl from "../../__testUtils/mockAccessControl";
 import { AccessControlProvider } from "../../src/contexts/accessControlContext";
 import mockPermissionsContextProvider from "../../__testUtils/mockPermissionsContextProvider";
-import useConsentBasedAccessForResource from "../../src/hooks/useConsentBasedAccessForResource";
+import useAccessBasedAccessForResource from "../../src/hooks/useAccessBasedAccessForResource";
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/index.test.jsx:36]: `useAccessGrantBasedAccessForResource`

File: components/resourceDetails/index.test.jsx [hunk 20]
```diff
@@ -33,28 +33,28 @@ import ResourceDetails, {
...
+jest.mock("../../src/hooks/useAccessBasedAccessForResource");
+const mockedUseAccessBasedAccessForResource = useAccessBasedAccessForResource;
 
 jest.mock("../../src/hooks/useAccessControlType");
 const mockUseAccessControlType = useAccessControlType;
 
-jest.mock("../../src/hooks/useConsentBasedAccessForResource");
-const mockUseConsentBasedAccessForResource = useConsentBasedAccessForResource;
+jest.mock("../../src/hooks/useAccessBasedAccessForResource");
+const mockUseAccessBasedAccessForResource = useAccessBasedAccessForResource;
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/index.test.jsx:50]: same as above

File: components/resourceDetails/index.test.jsx [hunk 21]
```diff
@@ -33,28 +33,28 @@ import ResourceDetails, {
...
 
 describe("Resource details", () => {
   beforeEach(() => {
     jest
       .spyOn(routerFns, "useRouter")
       .mockReturnValue({ query: { resourceIri: "" }, push: jest.fn() });
-    mockedUseConsentBasedAccessForResource.mockReturnValue([]);
+    mockedUseAccessBasedAccessForResource.mockReturnValue([]);
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/index.test.jsx:57]: same as above

File: components/resourceDetails/index.test.jsx [hunk 22]
```diff
@@ -109,7 +109,7 @@ describe("Resource details", () => {
 
   it("renders Sharing component for ACP-supporting Solid servers", async () => {
     const PermissionsContextProvider = mockPermissionsContextProvider();
-    mockUseConsentBasedAccessForResource.mockReturnValue({ permissions: [] });
+    mockUseAccessBasedAccessForResource.mockReturnValue({ permissions: [] });
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/index.test.jsx:112]: same as above

File: components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap [hunk 23]
```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
+exports[`AgentAccessTable with access based agents renders a list of permissions including access based agent 1`] = `
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap:3]: access grant based, in both occurrences in the sentence

File: components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap [hunk 24]
```diff
@@ -173,41 +173,7 @@ exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
                           class="PodBrowser-root PodBrowser-detailText PodBrowser-body1 PodBrowser-detailText"
                           data-testid="agent-web-id"
                         >
-                          Example B
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap:176]: this is incorrectly updated - we should see all the agents as we did before this was updated

File: components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap [hunk 25]
```diff
@@ -223,26 +189,6 @@ exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
                   </tr>
                 </tbody>
               </table>
-              <div
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap:226]: same as above

File: components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap [hunk 26]
```diff
@@ -252,7 +198,7 @@ exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
 </DocumentFragment>
 `;
 
-exports[`AgentAccessTable with consent based agents renders a list of permissions including consent based agent 1`] = `
+exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap:201]: `AgentAccessTable with access grants based agents renders a list of permissions including access grants  based agent `

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx [hunk 27]
```diff
@@ -125,7 +125,7 @@ const permissions = [
   },
 ];
 
-const permissionsWithConsentAgent = [
+const permissionsWithAccessAgent = [
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:128]: `permissionsWithAccessGrantBasedAccessAgent`

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx [hunk 28]
```diff
@@ -160,7 +160,7 @@ const permissionsWithProfilesNamelessAgent = [
   },
 ];
 
-const permissionsWithConsentWithProfiles = permissionsWithConsentAgent.map(
+const permissionsWithAccessWithProfiles = permissionsWithAccessAgent.map(
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:163]: `permissionsWithAccessGrantWithProfiles` - `permissionsWithAccessGrantAgent`

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx [hunk 29]
```diff
@@ -398,16 +398,16 @@ describe("AgentAccessTable with agents", () => {
     });
   });
 });
-describe("AgentAccessTable with consent based agents", () => {
+describe("AgentAccessTable with access based agents", () => {
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:401]: access grant based

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx [hunk 30]
```diff
@@ -398,16 +398,16 @@ describe("AgentAccessTable with agents", () => {
...
   const mockDataset = mockSolidDatasetFrom("https://example.org/resource");
   const setLoading = jest.fn();
   beforeEach(() => {
     mockedUsePermissionsWithProfiles.mockReturnValue({
-      permissionsWithProfiles: permissionsWithConsentWithProfiles,
+      permissionsWithProfiles: permissionsWithAccessWithProfiles,
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:406]: same as above

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx [hunk 31]
```diff
@@ -398,16 +398,16 @@ describe("AgentAccessTable with agents", () => {
...
     });
   });
 
-  it("renders a list of permissions including consent based agent", async () => {
+  it("renders a list of permissions including access based agent", async () => {
```
Comment (VirginiaBalseiro) [inline components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:410]: same as above

File: components/resourceDrawer/index.test.jsx [hunk 32]
```diff
@@ -37,15 +37,15 @@ import useAccessControl from "../../src/hooks/useAccessControl";
 import mockAccessControl from "../../__testUtils/mockAccessControl";
 import { ResourceInfoProvider } from "../../src/contexts/resourceInfoContext";
 import mockPermissionsContextProvider from "../../__testUtils/mockPermissionsContextProvider";
-import useConsentBasedAccessForResource from "../../src/hooks/useConsentBasedAccessForResource";
+import useAccessBasedAccessForResource from "../../src/hooks/useAccessBasedAccessForResource";
```
Comment (VirginiaBalseiro) [inline components/resourceDrawer/index.test.jsx:40]: same as before

File: components/resourceDrawer/index.test.jsx [hunk 33]
```diff
@@ -37,15 +37,15 @@ import useAccessControl from "../../src/hooks/useAccessControl";
...
 import usePermissionsWithProfiles from "../../src/hooks/usePermissionsWithProfiles";
 import useAllPermissions from "../../src/hooks/useAllPermissions";
 
 jest.mock("../../src/hooks/useResourceInfo");
 const mockUseResourceInfo = useResourceInfo;
 jest.mock("../../src/hooks/useAccessControl");
-jest.mock("../../src/hooks/useConsentBasedAccessForResource");
-const mockUseConsentBasedAccessForResource = useConsentBasedAccessForResource;
+jest.mock("../../src/hooks/useAccessBasedAccessForResource");
```
Comment (VirginiaBalseiro) [inline components/resourceDrawer/index.test.jsx:47]: same as before

File: components/resourceDrawer/index.test.jsx [hunk 34]
```diff
@@ -37,15 +37,15 @@ import useAccessControl from "../../src/hooks/useAccessControl";
...
+const mockUseAccessBasedAccessForResource = useAccessBasedAccessForResource;
```
Comment (VirginiaBalseiro) [inline components/resourceDrawer/index.test.jsx:48]: same as before

File: components/resourceDrawer/index.test.jsx [hunk 35]
```diff
@@ -87,7 +87,7 @@ describe("ResourceDrawer view", () => {
       accessControl: mockAccessControl(),
       accessControlType: "acp",
     };
-    mockUseConsentBasedAccessForResource.mockReturnValue({ permissions: [] });
+    mockUseAccessBasedAccessForResource.mockReturnValue({ permissions: [] });
```
Comment (VirginiaBalseiro) [inline components/resourceDrawer/index.test.jsx:90]: same as before

File: constants/policies.js [hunk 36]
```diff
@@ -101,7 +101,7 @@ export const POLICIES_TYPE_MAP = {
     iconClassName: "iconEditor",
     titlePlural: "Editors",
     titleSingular: "Editor",
-    titleConsent: "wants to create, edit, or delete",
+    titleAccess: "wants to create, edit, or delete",
```
Comment (VirginiaBalseiro) [inline constants/policies.js:104]: `titleAccessGrant` (same as in another file)

File: constants/policies.js [hunk 37]
```diff
@@ -119,7 +119,7 @@ export const POLICIES_TYPE_MAP = {
     iconClassName: "iconViewer",
     titlePlural: "Viewers",
     titleSingular: "Viewer",
-    titleConsent: "wants to see",
+    titleAccess: "wants to see",
```
Comment (VirginiaBalseiro) [inline constants/policies.js:122]: same as above

File: constants/policies.js [hunk 38]
```diff
@@ -156,7 +156,7 @@ export const POLICIES_TYPE_MAP = {
     title: "Edit Only",
     titlePlural: "Edit Only",
     titleSingular: "Edit Only",
-    titleConsent: "wants to edit",
+    titleAccess: "wants to edit",
```
Comment (VirginiaBalseiro) [inline constants/policies.js:159]: same as above

File: constants/policies.js [hunk 39]
```diff
@@ -167,7 +167,7 @@ export const POLICIES_TYPE_MAP = {
     title: "Add Only",
     titlePlural: "Add Only",
     titleSingular: "Add Only",
-    titleConsent: "wants to add to",
+    titleAccess: "wants to add to",
```
Comment (VirginiaBalseiro) [inline constants/policies.js:170]: same as above

File: pages/privacy/access/requests/index.jsx [hunk 40]
```diff
@@ -21,12 +21,12 @@
 
 import React from "react";
 import AuthenticatedRoute from "../../../../src/authentication/AuthenticatedRoute";
-import ConsentShowPage from "../../../../components/pages/privacy/access/requests";
+import AccessShowPage from "../../../../components/pages/privacy/access/requests";
```
Comment (VirginiaBalseiro) [inline pages/privacy/access/requests/index.jsx:24]: `AccessRequestShowPage`

File: pages/privacy/access/requests/index.jsx [hunk 41]
```diff
@@ -21,12 +21,12 @@
...
 
-export default function ConsentShow() {
+export default function AccessShow() {
```
Comment (VirginiaBalseiro) [inline pages/privacy/access/requests/index.jsx:26]: `AccessRequestShow`

File: pages/privacy/access/requests/index.jsx [hunk 42]
```diff
@@ -21,12 +21,12 @@
...
   return (
     <AuthenticatedRoute>
-      <ConsentShowPage />
+      <AccessShowPage />
```
Comment (VirginiaBalseiro) [inline pages/privacy/access/requests/index.jsx:29]: same as above

File: src/accessControl/acp/index.js [hunk 43]
```diff
@@ -60,8 +60,8 @@ import { ACP_TYPE_MAP, POLICIES_TYPE_MAP } from "../../../constants/policies";
 export const noAcrAccessError =
   "No access to Access Control Resource for this resource";
 
-const CONTROL_ACCESS_CONSENT = {
-  titleConsent: "wants to add or remove people from",
+const CONTROL_ACCESS_ACCESS = {
+  titleAccess: "wants to add or remove people from",
```
Comment (VirginiaBalseiro) [inline src/accessControl/acp/index.js:64]: same as above

File: src/hooks/useAccessBasedAccessForResource/index.test.jsx [hunk 44]
```diff
@@ -20,8 +20,8 @@
  */
 
 import { renderHook } from "@testing-library/react-hooks";
-import * as consentFns from "@inrupt/solid-client-access-grants";
-import useConsentBasedAccessForResource from "./index";
+import * as accessFns from "@inrupt/solid-client-access-grants";
+import useAccessBasedAccessForResource from "./index";
```
Comment (VirginiaBalseiro) [inline src/hooks/useAccessBasedAccessForResource/index.test.jsx:24]: same as before

File: src/hooks/useAllPermissions/index.test.jsx [hunk 45]
```diff
@@ -55,15 +55,15 @@ describe("useAllPermissions", () => {
   };
 
   it("returns an empty array if no access control", async () => {
-    mockedUseConsentBasedAccessForResource.mockReturnValue({
+    mockedUseAccessBasedAccessForResource.mockReturnValue({
```
Comment (VirginiaBalseiro) [inline src/hooks/useAllPermissions/index.test.jsx:58]: same here

File: src/models/access/signedVc/index.js [hunk 46]
```diff
@@ -20,17 +20,17 @@
  */
 
 export function getRequestedAccessesFromSignedVc(signedVc) {
-  return Array.isArray(signedVc?.credentialSubject?.providedConsent.mode)
-    ? signedVc?.credentialSubject?.providedConsent
-    : [signedVc?.credentialSubject?.providedConsent];
+  return Array.isArray(signedVc?.credentialSubject?.providedAccess.mode)
+    ? signedVc?.credentialSubject?.providedAccess
```
Comment (VirginiaBalseiro) [inline src/models/access/signedVc/index.js:24]: same here and the line below

File: src/models/access/signedVc/index.js [hunk 47]
```diff
@@ -20,17 +20,17 @@
...
+    : [signedVc?.credentialSubject?.providedAccess];
 }
 
 export function getRequestorWebIdFromSignedVc(signedVc) {
-  return signedVc?.credentialSubject?.providedConsent?.isProvidedTo;
+  return signedVc?.credentialSubject?.providedAccess?.isProvidedTo;
```
Comment (VirginiaBalseiro) [inline src/models/access/signedVc/index.js:29]: same here

File: src/models/access/signedVc/index.js [hunk 48]
```diff
@@ -20,17 +20,17 @@
...
 }
 
 export function getPurposeUrlsFromSignedVc(signedVc) {
-  return Array.isArray(signedVc?.credentialSubject?.providedConsent)
-    ? signedVc?.credentialSubject?.providedConsent[0].forPurpose // getting first item in array for now
-    : signedVc?.credentialSubject?.providedConsent.forPurpose;
+  return Array.isArray(signedVc?.credentialSubject?.providedAccess)
```
Comment (VirginiaBalseiro) [inline src/models/access/signedVc/index.js:33]: same here and the lines below

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]

File: __testUtils/mockSignedVc.js [hunk 49]
```diff
@@ -27,7 +27,7 @@ export default function getSignedVc() {
       "https://consent.pod.inrupt.com/credentials/v1",
     ],
     credentialSubject: {
-      providedConsent: {
+      providedAccess: {
```
[the remark this answers is not in the document]
Comment (solid-akb) [reply __testUtils/mockSignedVc.js:30]: done

File: components/accessRequestForm/__snapshots__/index.test.jsx.snap [hunk 50]
```diff
@@ -290,8 +187,9 @@ exports[`Consent Request Form Renders a consent request form with multiple purpo
               >
                 <input
                   class="PodBrowser-input PodBrowser-input"
+                  data-testid="access-access-switch"
                   type="checkbox"
-                  value="https://pod.inrupt.com/alice/private/data"
+                  value="https://pod.inrupt.com/alice/private/data/"
```
[the remark this answers is not in the document]
Comment (solid-akb) [reply components/accessRequestForm/__snapshots__/index.test.jsx.snap:192]: I actually don't know where this is coming from - I did not change the resource to a directory but it's only changing in the snapshots. I looked at the history and I couldn't see any changes recently to the Alice resource so at a loss as to why this is happening.

[the remark this answers is not in the document]
Comment (VirginiaBalseiro) [reply components/accessRequestForm/__snapshots__/index.test.jsx.snap:192]: @solid-akb ok - thought it was intentional. Just ignore then.

File: components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap (hunk 26 above)
Comment (VirginiaBalseiro) [reply components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap:201]: That's the name of the test. You need to change the title in the test and then update the snapshot (I might have missed this in the actual test file)

File: components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx (hunk 27 above)
Comment (solid-akb) [reply components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx:128]: This actually isn't being used anywhere so I took it out.

File: components/accessRequestForm/__snapshots__/index.test.jsx.snap [hunk 51]
```diff
@@ -17,110 +17,7 @@ exports[`Consent Request Form Renders a consent request form with multiple purpo
     <div
       class="PodBrowser-purposes-container"
     >
-      Select the purposes you wish to allow:
```
[the remark this answers is not in the document]
Comment (VirginiaBalseiro) [reply components/accessRequestForm/__snapshots__/index.test.jsx.snap:20]: I noticed that tis is just that the tests got switched around

**Review by ThisIsMissEm — APPROVED** [review]
Looks good to me / a straight forward find & replace

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
When not selecting any access, the dialog displayed is incorrect:

image

it should display: 

image

I left a suggestion that should fix it.

File: components/accessRequestForm/index.jsx [hunk 52]
```diff
@@ -149,15 +148,15 @@ export default function ConsentRequestForm({
...
       setConfirmationSetup(true);
-      setOpen(CONSENT_REQUEST_NO_ACCESS_DIALOG);
+      setOpen(ACCESS_REQUEST_NO_ACCESS_DIALOG);
       setTitle(NO_PURPOSE_TITLE);
       setCancelText("Ok");
       setOmitConfirmButton(true);
       setContent(NO_PURPOSE_CONTENT);
     } else if (!selectedAccess.length) {
       setConfirmationSetup(true);
       setOmitCancelButton(false);
```
Comment (VirginiaBalseiro) [inline components/accessRequestForm/index.jsx:158]:
```suggestion
      setOmitCancelButton(false);
      setOmitConfirmButton(false);
```

Comment (VirginiaBalseiro) [reply components/accessRequestForm/index.jsx:158]: This should fix the confirmation dialog bug.

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]

**Review by VirginiaBalseiro — APPROVED** [review]

**Pull request merged**